Задания с 28.09.2021/28.09.2021-Task1/28.09.2021-Task1/BinaryRepresentation.c

```diff
@@ -0,0 +1,189 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <stdbool.h>
+#include <math.h>
+#define INT_BITS_COUNT sizeof(int) * 8
```

Comment:
```suggestion
#include <math.h>

#define INT_BITS_COUNT sizeof(int) * 8
```
Лучше разделять секции файла пустыми строками